### PR TITLE
Add help page keys for more settings tabs

### PR DESCRIPTION
**Pull request by nyalldawson**

### CODE REVIEW

**Comment by nyalldawson** [comment]
Att @DelazJ

File: src/app/options/qgscustomprojectionoptions.cpp [hunk 1]
```diff
@@ -420,7 +420,7 @@ QString QgsCustomProjectionOptionsWidget::multiLineWktToSingleLine( const QStrin
 
 QString QgsCustomProjectionOptionsWidget::helpKey() const
 {
-  return QStringLiteral( "working_with_projections/working_with_projections.html" );
+  return QStringLiteral( "introduction/qgis_configuration.html#user-defined-crs" );
```
Comment (DelazJ) [inline src/app/options/qgscustomprojectionoptions.cpp:423]: I'm divided here. The https://docs.qgis.org/3.34/en/docs/user_manual/working_with_projections/working_with_projections.html#custom-coordinate-reference-system is somewhat more descriptive (and a bit of duplicate afaics --- we should probably keep only one detailed section)

Comment (nyalldawson) [reply src/app/options/qgscustomprojectionoptions.cpp:423]: Ok, I agree that that's nicer. I think at least `introduction/qgis_configuration.html#user-defined-crs` should include a link to this extra info!

Comment (nyalldawson) [inline src/app/options/qgscustomprojectionoptions.cpp:423]:
```suggestion
  return QStringLiteral( "working_with_projections/working_with_projections.html" );
```

Comment (nyalldawson) [reply src/app/options/qgscustomprojectionoptions.cpp]: I've reverted it now

**Pull request closed without merging**